nationality-park/park.js

```diff
@@ -0,0 +1,50 @@
+import parksData from "./nationalParksOfPolandData.json" assert { type: "json" };
+
+const queryString = window.location.search;
+const urlParams = new URLSearchParams(queryString);
+const name = urlParams.get("parkname");
+
+if (name == null) {
+  window.location.href = "index.html";
+}
+window.addEventListener("DOMContentLoaded", () => {
+  displayData();
+});
+
+const displayData = () => {
+  const parkData = parksData.filter((element) => element.name == name);
```

Comment:
1. use strong comparison, 
2. check how "find" method works and use it, 
3. destructure data